addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.scss

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_AttachmentLinkPreview_linkImage {
+    max-width: 160px;
+    max-height: 160px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_AttachmentLinkPreview_asideItemUnlink {
+    background: transparent;
+}
+
+.o_AttachmentLinkPreview {
+    background-color: gray('300');
+}
```

Comment:
use equivalent bootstrap classnames instead, here `bg-transparent` and `bg-300` I think